build/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM golang:1.5
+
+MAINTAINER Peter Mattis <peter@cockroachlabs.com>
+
+RUN curl --silent --location https://deb.nodesource.com/setup_4.x | bash - && \
+ apt-get dist-upgrade -y && \
+ apt-get install --no-install-recommends --auto-remove -y git build-essential file nodejs && \
+ apt-get clean autoclean && \
+ apt-get autoremove -y && \
+ rm -rf /tmp/*
+
+COPY bower.json npm-shrinkwrap.json package.json tsd.json Makefile ./
+
+RUN make bower.installed npm.installed tsd.installed && \
+ rm bower.json npm-shrinkwrap.json package.json tsd.json Makefile && \
```

Comment:
I was thinking that a mismatch between the builder version on the repo version would cause `circle-deps.sh` to fail. That would get people's attention very fast.

Are you sure new deps will get downloaded if there is a mismatch? Perhaps I don't understand what the code at `circle-deps.sh:15` is doing.